exports/taskfiles/utils/boost.yaml

```diff
@@ -0,0 +1,166 @@
+version: "3"
+
+includes:
+  remote: "remote.yaml"
+
+set: ["u", "pipefail"]
+shopt: ["globstar"]
+
+tasks:
+  # Runs the bootstrap.sh generate step in the given source directory. Boost only supports
+  # in-source generation and building.
+  #
+  # @param {string} SOURCE_DIR Project source directory.
+  # @param {string} INSTALL_PREFIX Path prefix of where the project should be installed.
+  # @param {string[]} TARGETS Target libraries to build.
+  # @param {string[]} [EXTRA_ARGS] Any additional arguments to pass to the generate command.
+  generate:
+    internal: true
+    dir: "{{.SOURCE_DIR}}"
+    cmds:
+      - >-
+        ./bootstrap.sh
+        --prefix="{{.INSTALL_PREFIX}}"
+        --exec-prefix="{{.INSTALL_PREFIX}}"
+        --with-libraries={{(join "," .TARGETS)}}
+        {{- range .EXTRA_ARGS}}
+        "{{.}}"
+        {{- end}}
```

Comment:
This task and others are missing `requires` field and also setting default for optional lists. See other tasks in utils.
Unfortunately, I probably missed this in the spider PR.